src/eckit/geo/cache/Unzip.cc

```diff
@@ -10,6 +10,9 @@
  */
 
 
+#include <algorithm>
+#include <memory>
```

Comment:
Missing includes, but not related to condo (just coincidence), rather but just generally missing. Perhaps the PR title could be amended.

For consistency, perhaps they should also be guarded by `eckit_HAVE_ZIP`, just like <array> and <fstream> below.